styles.css

```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.header-logo {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 0.1%;
+  align-items: center;
+  background-color: #fff;
+}
+
+.logo {
+  height: 1.688rem;
+  width: 8.125rem;
+}
+
+.image-icon {
+  height: 3.438rem;
+  width: 3.125rem;
+  margin-top: 1%;
+  margin-right: 11%;
+  margin-bottom: 0;
+  color: #6070ff;
+}
+
+.title {
+  font-family: 'Poppins', sans-serif;
+  font-weight: bold;
+  font-size: 3.125rem;
+  line-height: 3.25rem;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-bottom: 2rem;
+}
+
+.intro {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 6.25rem;
+  font-size: 16px;
+  line-height: 1.5rem;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-bottom: 20px;
+}
+
+.lets-connect {
+  width: 9.375rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 110%;
+  line-height: 1.5rem;
+  color: #7f8cff;
+  display: flex;
+  order: 0;
+  flex-grow: 0;
+}
+
+.social-buttons {
+  display: flex;
+  margin-top: 4%;
+  margin-right: 6.25rem;
+  list-style-type: none;
+  align-items: center;
+}
+
+.social-buttons li {
+  margin-right: 12px;
+}
+
+.icons {
+  border-left: 100px;
+  margin-left: 1%;
+  margin-right: 4%;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.all-body {
+  padding: 7rem 1.5rem;
+  border-radius: 0 0 0 80px;
+  background-color: white;
+  overflow: hidden;
+  height: 430px;
+  z-index: -1;
+  position: relative;
+}
```

Comment:
> - [ ] Great work so far!👍 I suggest using Flexbox to align this content. Flexbox provides a range of options to align all of our elements, and it is also responsive as stated in the project requirement.☺
>
> - _In order to lay out the elements on the page you should use flexbox in all 2 sections._
> 

> - [ ] Please make sure that the paddings align with the design in Figma.👍
> 
> Figma Design👇
> 
> ![padding1](https://user-images.githubusercontent.com/40009354/234958368-379b281a-afbd-4e5b-837c-31a3bbcfda1d.PNG)
> 
> Your Design👇
> 
> ![padding](https://user-images.githubusercontent.com/40009354/234958017-ab0ddeb3-c6fb-4f2f-a48f-48864bf14ac5.png)
>

Reply:
- Hey, as suggested by the previous reviewer, this main container should be aligned using Flexbox, while you've used flex on the children element, the parent should have the display property of Flex as this is the correct way to give the immediate children to inherit it from the parent. It shouldn't have a position of relative but rather flex. let's update this part of the design.